### PR TITLE
Translations

### LINKED ISSUE #6: Make the app translatable
We should make the app translatable finally

### PR DESCRIPTION
**Pull request by NathanBnm**
Here is the PR that makes your app translatable. I created different commits for more clarity :smile:
This fixes #6.

Note: I'll be able to add French translations then.

**This PR is ready for review**

### CODE REVIEW

File: src/Widgets/ToolBars/ImageToolBar.vala [hunk 1]
```diff
@@ -30,13 +30,13 @@ namespace Writer.Widgets {
             get_style_context ().add_class ("writer-toolbar");
 
             var wrap_combobox = new Gtk.ComboBoxText ();
-            wrap_combobox.append ("In line of text", ("In line of text"));
-            wrap_combobox.append ("Float above text", ("Float above text"));
+            wrap_combobox.append (_("In line of text"), ("In line of text"));
+            wrap_combobox.append (_("Float above text"), ("Float above text"));
```
Comment (ryonakano) [inline src/Widgets/ToolBars/ImageToolBar.vala]:
```suggestion
            wrap_combobox.append ("Float above text"), _("Float above text"));
```

File: src/Widgets/ToolBars/ImageToolBar.vala [hunk 2]
```diff
@@ -30,13 +30,13 @@ namespace Writer.Widgets {
             get_style_context ().add_class ("writer-toolbar");
 
             var wrap_combobox = new Gtk.ComboBoxText ();
-            wrap_combobox.append ("In line of text", ("In line of text"));
-            wrap_combobox.append ("Float above text", ("Float above text"));
+            wrap_combobox.append (_("In line of text"), ("In line of text"));
```
Comment (ryonakano) [inline src/Widgets/ToolBars/ImageToolBar.vala]:
```suggestion
            wrap_combobox.append ("In line of text", _("In line of text"));
```

The first argument of `append ()` method of `Gtk.ComboBoxTest` is an id string to distinguish which text currently selected, so it should not be translatable. Instead, the second argument is shown in UI and should be translatable.
See https://valadoc.org/gtk+-3.0/Gtk.ComboBoxText.append.html for more details.

File: src/Widgets/ToolBars/TextToolBar.vala [hunk 3]
```diff
@@ -41,12 +41,12 @@ namespace Writer.Widgets {
             editor.cursor_moved.connect (cursor_moved);
 
             var paragraph_combobox = new Gtk.ComboBoxText ();
-            paragraph_combobox.append ("Paragraph", ("Paragraph"));
-            paragraph_combobox.append ("Title", ("Title"));
-            paragraph_combobox.append ("Subtitle", ("Subtitle"));
-            paragraph_combobox.append ("Bullet List", ("Bullet List"));
-            paragraph_combobox.append ("Numbered List", ("Numbered List"));
-            paragraph_combobox.append ("Two-Column", ("Two-Column"));
+            paragraph_combobox.append (_("Paragraph"), ("Paragraph"));
```
Comment (ryonakano) [inline src/Widgets/ToolBars/TextToolBar.vala]:
```suggestion
            paragraph_combobox.append ("Paragraph", _("Paragraph"));
```

File: src/Widgets/ToolBars/TextToolBar.vala [hunk 4]
```diff
@@ -41,12 +41,12 @@ namespace Writer.Widgets {
...
+            paragraph_combobox.append (_("Title"), ("Title"));
```
Comment (ryonakano) [inline src/Widgets/ToolBars/TextToolBar.vala]:
```suggestion
            paragraph_combobox.append ("Title", _("Title"));
```

File: src/Widgets/ToolBars/TextToolBar.vala [hunk 5]
```diff
@@ -41,12 +41,12 @@ namespace Writer.Widgets {
...
+            paragraph_combobox.append (_("Subtitle"), ("Subtitle"));
```
Comment (ryonakano) [inline src/Widgets/ToolBars/TextToolBar.vala]:
```suggestion
            paragraph_combobox.append ("Subtitle", _("Subtitle"));
```

File: src/Widgets/ToolBars/TextToolBar.vala [hunk 6]
```diff
@@ -41,12 +41,12 @@ namespace Writer.Widgets {
...
+            paragraph_combobox.append (_("Bullet List"), ("Bullet List"));
```
Comment (ryonakano) [inline src/Widgets/ToolBars/TextToolBar.vala]:
```suggestion
            paragraph_combobox.append ("Bullet List", _("Bullet List"));
```

File: src/Widgets/ToolBars/TextToolBar.vala [hunk 7]
```diff
@@ -41,12 +41,12 @@ namespace Writer.Widgets {
...
+            paragraph_combobox.append (_("Numbered List"), ("Numbered List"));
```
Comment (ryonakano) [inline src/Widgets/ToolBars/TextToolBar.vala]:
```suggestion
            paragraph_combobox.append ("Numbered List", _("Numbered List"));
```

File: src/Widgets/ToolBars/TextToolBar.vala [hunk 8]
```diff
@@ -41,12 +41,12 @@ namespace Writer.Widgets {
...
+            paragraph_combobox.append (_("Two-Column"), ("Two-Column"));
```
Comment (ryonakano) [inline src/Widgets/ToolBars/TextToolBar.vala]:
```suggestion
            paragraph_combobox.append ("Two-Column", _("Two-Column"));
```

File: src/Writer.vala [hunk 9]
```diff
@@ -110,7 +110,7 @@ namespace Writer {
             File? file = null;
 
             do {
-                file_name = "Untitled Document %i".printf (id++);
+                file_name = _("Untitled Document") + "%i".printf (id++);
```
Comment (ryonakano) [inline src/Writer.vala]:
```suggestion
                file_name = _("Untitled Document %i").printf (id++);
```

I don't know specifically what language has this construction, but I suppose some language may need to bring numbers at the beginning of translation. Including the string `%s` (called "string format specifier") into translations allows translators to bring the number to any position of their translation.

**Review by ryonakano — CHANGES_REQUESTED** [review]
Just a few tiny things.

**Comment by ryonakano** [comment]
Addressing these changes modifies some strings, so please update .po and .pot files again. :pray:

File: src/Widgets/ToolBars/ImageToolBar.vala (hunk 2 above)
Comment (NathanBnm) [reply src/Widgets/ToolBars/ImageToolBar.vala]: I wasn't sure about which one was the string that should be translated thanks for your explanation.

File: src/Writer.vala (hunk 9 above)
Comment (NathanBnm) [reply src/Writer.vala]: Yes you're completely right!

**Comment by NathanBnm** [comment]
> Addressing these changes modifies some strings, so please update .po and .pot files again. pray

Everything should be okay now :smile:

**Review by ryonakano — APPROVED** [review]
Thank you for your branch!!

**Pull request merged**